themes/wilson_theme_starterkit/components/atoms/button/button.twig

```diff
@@ -0,0 +1,3 @@
+{% for item in items %}
+  <a href="{{ item.content['#url'] }}" class="btn {{'btn--' ~ type}} {{item.content['#url'].isExternal ? 'is-external'}}" {% if item.content['#url'].isExternal %} target="_blank"{% endif %}>{{ item.content['#title'] }}</a>
```

Comment:
We'll probably want to set more generic variables within storybook and pass the for loop back to the drupal field template (include this twig file each time it creates an item)

Reply:
Templates change